git_machete/cmd.py

```diff
@@ -2069,18 +2148,32 @@ def check_required_param(in_args: List[str], allowed_values_string: str) -> str:
             if dest != cb:
                 git.checkout(dest)
         elif cmd == "github":
-            github_allowed_subcommands = "anno-prs|create-pr|retarget-pr"
-            param = check_required_param(parse_options(args, "", ["draft"]), github_allowed_subcommands)
+            github_allowed_subcommands = "anno-prs|create-pr|retarget-pr|checkout-pr"
+            list_args = parse_options(args) if args[0] == 'checkout-pr' else check_required_param(parse_options(args, "", ["draft"]), github_allowed_subcommands)
```

Comment:
Whoops the logic isn't right here...

```
$ git machete github create-pr 

github requires a subcommand: one of anno-prs|create-pr|retarget-pr|checkout-pr
```

I'd strongly suggest providing tests for the existing logic first (before this PR is further developed) to avoid painful regressions :/